package.json

```diff
@@ -18,7 +19,6 @@
   },
   "devDependencies": {
     "browserify": "^11.0.1",
```

Comment:
though it's a bit odd `browserify` is a dev dependency and not production dependency, I understand folks likely already have installed their own version and we shouldn't pin people to our particular version (as long as it doesn't break compatibility with `browserify-css`)